packages/shell-chrome/views/_partials/footer.edge

```diff
@@ -0,0 +1,109 @@
+<div
+    :class="{
+    'hidden': !showTools,
+}"
+    class="flex font-bold text-gray-400 border-t border-gray-300 bg-white"
+>
+    <div class="flex-1">
+        <div class="flex items-center text-xs leading-9 font-medium font-mono">
+            <div class="flex-1 pl-3" x-text="footerText"></div>
+
+            @verbatim
+            <div
+                :class="{
+                        'hidden': !showTools,
+                        'bg-gray-300 bg-opacity-50 text-gray-600': !showTools,
```

Comment:
should we make this 1 statment?

```suggestion
                        'hidden bg-gray-300 bg-opacity-50 text-gray-600': !showTools,
```

We could also use x-show for the `hidden` bit

Reply:
Ah I've left in some code in the footer that shouldn't be there 😬 I've removed it and we now toggle the footer visibility with `x-show` 👍🏼